src/client/routes/BUILD.bazel

```diff
@@ -1,9 +1,17 @@
-load("//bazel/js:defaults.bzl", "build_module", "build_route", "generate_route_manifest", "pkg_web")
+load("@npm//webpack-cli:index.bzl", "webpack_cli")
+load("//bazel/js:defaults.bzl", "build_module", "pkg_web")
+load("@rules_spa//spa:defs.bzl", "build_route", "generate_route_manifest")
 
 build_route(
     name = "default",
-    data = ["default.tsx"],
+    srcs = ["default.tsx"],
+    data = [
+        "//:package.json",
+        "@npm//:node_modules",
+    ],
     entry = "./$(execpath default.tsx)",
+    shared = "//bazel/js/internals/webpack:webpack.module-federation.shared.js",
```

Comment:
“shared” doesn’t seem directly clear, maybe federation_shared_config or something more explicit on the attribute?

Reply:
Will rename upstream

Reply:
https://github.com/Aghassi/rules_spa/pull/10

Reply:
Need to regen the docs, on my M1 at the moment

Reply:
https://github.com/Aghassi/rules_spa/releases/tag/v0.0.7